go/runtime/bundle/manager.go

```diff
@@ -46,6 +47,14 @@ type ManifestStore interface {
 	// AddManifest adds the provided manifest, whose components were extracted
 	// to the specified directory, to the store.
 	AddManifest(manifest *Manifest, dir string) error
+
+	// RemoveManifest removes a manifest with provided hash.
+	//
+	// Returns a path to exploded bundle dir of the removed manifest.
+	RemoveManifest(hash hash.Hash) (string, error)
```

Comment:
I would never expect that removing a manifest returns a path to exploded dir. Can this be removed? I would expect this function to return `bool`.